docs/userguide/dev/sync/custom.md

```diff
@@ -19,26 +19,32 @@ synchronization primitives.
 inlines. Provided in the internal header file
 [umutex.h](http://bugs.icu-project.org/trac/browser/icu/trunk/source/common/umutex.h).
```

Comment:
This should probably point to the file in the git repo.
FWIW, I'd probably suggest using a relative path like this:
https://github.com/unicode-org/icu/pull/948/files#diff-9201500aa8b7dc62e5adb5226ba04cc0R319
:)

Reply:
Thanks for catching this :)

Reply:
Thanks for making this PR! :)